.estafette.yaml

```diff
@@ -26,7 +26,7 @@ stages:
     image: extensions/docker:dev
     action: build
     inline: |
-      FROM gcr.io/google.com/cloudsdktool/cloud-sdk:405.0.0-alpine
+      FROM gcr.io/google.com/cloudsdktool/cloud-sdk:410.0.0-alpine
```

Comment:
why dont use `418` instead? https://console.cloud.google.com/gcr/images/google.com:cloudsdktool/GLOBAL/cloud-sdk

Reply:
There is breaking changes with latest image,  gke-gcloud-auth-plugin is longer exist,  I will create another task for this

Reply:
fixed with the installation of the gcloud auth plugin